kernel/aster-nix/src/fs/ext2/impl_for_vfs/inode.rs

```diff
@@ -44,19 +45,19 @@ impl Inode for Ext2Inode {
     }
 
     fn atime(&self) -> Duration {
-        self.atime()
+        Duration::from(self.atime())
     }
 
     fn set_atime(&self, time: Duration) {
-        self.set_atime(time)
+        self.set_atime(UnixTime::from(time))
     }
 
     fn mtime(&self) -> Duration {
-        self.mtime()
+        Duration::from(self.mtime())
```

Comment:
`mtime` calls `mtime`. The two methods having the same name but return different types. Don't you think that this code is confusing?

Reply:
I think the `UnixTime` is just used in Ext2. The time used in OS is a `Duration`, which is more fine-grained. For example, the Ext4 will use `Duration` as the timestamp